presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -116,18 +116,6 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
         this(metadata, sqlParser, featuresConfig, false, exporter);
     }
 
-    @PostConstruct
-    public void initialize()
-    {
-        stats.export(exporter);
-    }
-
-    @PreDestroy
-    public void destroy()
-    {
-        stats.unexport(exporter);
-    }
```

Comment:
This is actually only rearranging object methods after constructor (see after the constructor)